docs/additional/internal_aspects_de.md

```diff
@@ -0,0 +1,313 @@
+# Interne Aspekte
+
+Dieser Abschnitt beschreibt Aspekte, die von internen Entwicklern der Cloudogu GmbH beachten werden müssen.
+Sie enthalten Anforderungen zur Qualitätssicherung oder zum Beispiel grundlegende Definitionen zu einem fertigen,
+finalen Dogu.
+Somit ergibt sich, dass diese Informationen für externe Entwickler nicht zwingend relevant sind.
+Allerdings können die Aspekte genutzt werden, um Qualitätsstandards zu definieren oder zu verbessern.
+
+## Qualitätssicherung
+
+Dogus können sowohl aus selbst entwickelter Software als auch aus gekapselter Fremdsoftware bestehen.
+Im Falle einer Selbstentwicklung sollten übliche Maßstäbe für Qualitätssicherung gelten, die in den folgenden
+Abschnitten genannt werden.
+Wenn ein Dogu Software von Drittanbietern einfängt, ist es häufig nicht sinnvoll, sämtliche Features des eigentlichen
+Tools zu testen.
+Der Testfokus wird dann auf die hinzugefügten Dogu-Features abgegrenzt, die typischerweise den Mehrwert eines Dogus
+darstellen.
+
+### Qualitätssicherung des Tools
+
+Die Qualitätssicherung liegt in der Verantwortung des Inhabers.
+Für unsere Software setzen wir die folgenden Aspekte für die Qualitätssicherung um:
+
+- Unit-Tests (80 % Code-Abdeckung)
+- Integrations-Tests
+- End-to-End-Tests
+- Werkzeuge für statische Analyse z. B. Reviewdog, Linter (Style, Smells)
+- SonarQube (Code-Abdeckung, Smells, Bugs, Sicherheitslücken)
```

Comment:
SonarQube also offers static code analysis, otherwise things like code coverage, smells, bugs would not be possible. Should we join these two lines?